src/gh-api-client.ts

```diff
@@ -0,0 +1,114 @@
+import * as httpm from "@actions/http-client";
+
+const userAgent = "GitHub multi-repository variant analysis action";
+
+export function getApiClient() {
+  return new httpm.HttpClient(userAgent, [], {
+    allowRetries: true,
+  });
+}
+
+interface InProgressProperties {
+  status: "in_progress";
+}
+
+interface SuccessProperties {
+  status: "succeeded";
+  sourceLocationPrefix: string;
```

Comment:
Should these properties be `source_location_prefix` etc.? This is an interface representing the API model, which uses snake care rather than camel case.

Reply:
Good point, we didn't check these against the API schema closely enough 😢 

I've checked now and they should be:
- `source_location_prefix`
- `result_count`
- `database_commit_sha`